### ISSUE TITLE
Add more fields to Anki export

### ISSUE THREAD

**Issue by carina** [rsimmons/voracious#20]
I'm really, really loving voracious and it has been very useful so far. One feature I would love to have is having more data available in the Anki export. https://github.com/rsimmons/voracious/issues/16 already exists but in an ideal scenario (for me), the export would include:
- The translation of the selected word (=the translated text that is being displayed in the popup for the selected word)
- The reading of the selected word
- If there is a second subtitle language: the entire sentence in the second language
- Translations for all other words in the sentence (super optional, I'm generating this another way already but I would still find it helpful for studying these cards later)

This would be especially helpful since I can't seem to copy/paste any content within the app so I can not just copy translations into the text fields when creating cards. I'm using Voracious-0.5.0-mac.zip

--------
An example:
Already existing fields:
**Text:** 解放すると約束したろ
**Text with kanji readings:** 解放[かいほう]すると 約束[やくそく]したろ
**Selected word:** 解放 

New fields that would be super nice to have:
**Text (second language):** You promised you'd let me go
**Selected word meaning:** release; unleashing; liberation; emancipation; setting free; (comp) deallocation (of computer memory); 
**All meanings:** 
すると - (conj) thereupon; hereupon; (P)
かいほう - (n,vs) release; unleashing; liberation; emancipation; setting free; (comp) deallocation (of computer memory); (P)
やくそく - (n,vs) arrangement; promise; appointment; pact; engagement; convention; rule; (P)

(The example "all meanings" text was generated with a modified version of https://github.com/hsperr/japanese_meanings As you can see it is not perfect but most of the time it is very convenient to have)

**Comment by rsimmons** [comment]
Thanks for the detailed suggestions!

**Comment by carina** [comment]
I realized I forgot to mention one more field that would be useful: when dealing with verbs, it would be great to have the **dictionary form of the verb** (which is also displayed in the popup) available as a field instead of the conjugated form as it appeared in the subtitle.

I'm using this app almost daily and I love it. It has almost entirely replaced subs2srs for me (finally not having to use a windows VM anymore is great... ). It's a bit more effort at the moment since there are a couple of fields I have to fill in manually (especially since copy/paste does not work) but even in its current state it is really useful. Thanks again!

**Comment by chriskempson** [comment]
Would really love dictionary form too since I'm always grabbing that from the popup. 

Recently, I've been experimenting with exporting the English (second language) too. It's hidden on my cards by default and can be revealed with a click. Usually I try to steer clear of English but actually, there are times when a second opinion helps you grok a Japanese sentence.

**Comment by AnneUK** [comment]
Yes I just installed Voracious and I am really surprised that there is no separate export field at least for both languages (which I would expect in any app). Is it because of the idea of going monolingual asap :-D - interesting. As I am using the app for Korean I guess I can't really comment on the other fields.
